README.md

```diff
@@ -1 +1,43 @@
-# Kaidan
+# Kaidan - Cross platform XMPP client
+
+Installation
+------------
+
+On Linux do the following:
+
+Create a working directory
+
+ * mkdir src
+ * cd src
+
+Fetch swift source
+
+ * wget https://swift.im/downloads/releases/swift-3.0/swift-3.0.tar.gz
+ * tar -xzvf swift-3.0.tar.gz
+ * cd swift-3.0/
+
+Install all dependencies to build swiften
+
+ * ./BuildTools/InstallSwiftDependencies.sh
+ * ./scons Swiften -j<Number of threads>
+
+Install dependencies to build Kaidan (example for Debian)
+
+ * sudo apt-get install zlib1g-dev libssl-dev libxml2-dev libstdc++-4.9-dev libqt5quick5 libqt5quickparticles5 libqt5quickwidgets5 libqt5qml5 libqt5network5 libqt5gui5 libqt5core5a qt5-default libglib2.0-dev libpthread-stubs0-dev
```

Comment:
Debian 8 is the current stable. So I will refer to libstdc++-4.9-dev. Of course one can use apt install. I prefer to use apt-get.

-lz means libz.so which is in package zlib1g-dev. Try 
> dpkg -L zlib1g-dev.

-lSwiften_Boost means libSwiften_Boost.a. Try
> find <SWIFTEN3PATH> -name "*Swiften_Boost*"
-> ./3rdParty/Boost/libSwiften_Boost.a

I don't know what went wrong on your system. But you are a developer. I am sure you can figure out what is the cause ;-).